racun-service/src/main/java/rs/edu/raf/banka/racun/service/impl/UgovorService.java

```diff
@@ -167,7 +256,7 @@ public boolean finalizeUgovor(Long id, MultipartFile file) throws Exception {
         return true;
     }
 
-    public TransakcionaStavka addStavka(TransakcionaStavkaCreateRequest request) throws Exception {
+    public TransakcionaStavka addStavka(TransakcionaStavkaCreateRequest request, String token) throws Exception {
```

Comment:
Ovde treba proveriti da li treba token, ja bih rekao da ne treba jer mislim da stavku u ugovoru kreira onaj korisnik koji je kreirao i taj ugovor, a u ugovoru imamo userId (sada je to agentId, ali treba izmeniti).

Reply:
Treba da ima token:

1. Trebali bi da sačuvamo koje kreirao stavku
2. Treba da imamo validaciju da li stavku kreira onaj ko može da pristupi ugovoru. Preko fronta, ako ne može da pristupi ugovoru neće moći ni da doda stavku, ali problem je ako se API koristi van fronta.